src/effects/effectchainslot.h

```diff
@@ -34,6 +34,9 @@ class EffectChainSlot : public QObject {
 
     EffectSlotPointer addEffectSlot(const QString& group);
```

Comment:
make protected

Reply:
You can make this a separate commit then use `git rebase -i` to squash it into the EffectRack removal commit by marking it as "fixup" commit.